packages/ifpack2/src/Ifpack2_Details_ScaledDampedResidual_decl.hpp

```diff
@@ -111,6 +111,9 @@ class ScaledDampedResidual {
   std::unique_ptr<vector_type> X_colMap_;
   std::unique_ptr<multivector_type> V1_;
 
+  typename multivector_type::dual_view_type::t_host viewW_, viewB_, viewX_;
```

Comment:
@cgcgcg It's better not to store the Views.  (You were there for that part of my presentation Thursday morning, right? ;-) )  Here's why:

1. You only ever use the Views to check for aliasing, and you only ever use the raw pointer (result of `.data()`).

2. Storing the Views means you have to remember to update them if you ever need to resize or replace the Vectors.

Furthermore, you're storing rank-2 Views even though you're always using Vectors.

Reply:
@mhoemmen What's the correct way then to check whether two MultiVectors are identical? There is no `operator==`, right?

Reply:
@cgcgcg Don't store the Views.  Just get them temporarily where you need them.  Don't store them in the class.